extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/OidcTenantConfig.java

```diff
@@ -119,6 +125,40 @@
 
     }
 
+    @ConfigGroup
+    public static class Logout {
+
```

Comment:
I believe we need to support the OIDCs without the discovery (as we already do with `itrospection_path` and `jwks_path`), so adding `logout_path` and checking it if no discovery is available would make sense IMHO